src/components/AvailableNetworks.js

```diff
@@ -15,16 +15,17 @@ import { ETH_ADDRESS, ETH_SYMBOL } from '@/references';
 import Routes from '@/navigation/routesNames';
 import { padding, position } from '@/styles';
 import { ethereumUtils } from '@/utils';
+import { useTheme } from '@/theme';
 
-const AvailableNetworks = ({
```

Comment:
let's move this component under the `/components/exchange` folder to have both in the same place, WDYT?

Reply:
added to expanded state since that where its being used